internal/pkg/cli/svc_deploy.go

```diff
@@ -386,15 +386,15 @@ func (o *deploySvcOpts) configureClients() error {
 	return nil
 }
 
-func (o *deploySvcOpts) publicCIDRBlocks() ([]string, error) {
-	envDescription, err := o.envDescriber.Describe()
+func publicCIDRBlocks(envDescriber envDescriber, subnetLister vpcSubnetLister, envName string) ([]string, error) {
```

Comment:
The function name and parameters look a little bit strange as `publicCIDRBlocks` is just too bottom layer as a function in `cli` package, and `envDescriber` is already inited with the env name. Can we move [this whole code block](https://github.com/aws/copilot-cli/pull/3171/files#diff-887bb70e74d71d9ecbfe078d768a70bdd2f02043501ba27c8b254ac3a102e799R636-R668) to its own function instead of having `publicCIDRBlocks` in `svc_deploy`?